### PR TITLE
moneydoor

### PR DESCRIPTION
**Pull request by DXKopa**
moneydoor

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
+    private int ONE = 1;
+    private int TEN = 10;
+
+    // The game framework
+    private GameFramework framework = null;
+
+    // The amount of money each player has
+    public HashMap<String, Integer> PlayerScores = new HashMap<String, Integer>();
```
Comment (ghost) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java]: make this field private and provide accessors if necessary

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    public boolean isGameOver() {
+        //Iterate through the grid and see if any doors are still opened.
+        for(int i = 0;i < 10;i++) {
+            if (framework.getSquare(i, 0) == "Door closed!") {
+                return false;
+            }
+        }
+//System.out.println("game is over");
```
Comment (DXKopa) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java]: commented out code should be removed

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
```
Comment (ghost) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java]: expand star imports

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java (hunk 2 above)
Comment (ghost) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java]: Remove dead code here

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
looks good to me

**Pull request merged**